test-scripts/0.1.0/tests/deduction.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Thanks for creating these. Since we've already released 0.1.0 could you create a `test-scripts/0.2.0/test/` folder and move these files there? Don't worry about the other scripts and docker-compose we'll add those when the release is prepared.